DevDynamo.Web/Areas/ApiV1/Controllers/SystemController.cs

```diff
@@ -0,0 +1,34 @@
+using DevDynamo.Models;
+using DevDynamo.Web.Areas.ApiV1.Models;
+using DevDynamo.Web.Data;
+using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Mvc;
+using System.Reflection;
+
+namespace DevDynamo.Web.Areas.ApiV1.Controllers
+{
+    [Route("api/v1/[controller]")]
+    [ApiController]
+    public class SystemController : AppControllerBase
+    {
+        [HttpGet]
+        public ActionResult<SystemResponse> GetVersionSystem()
+        {
+         
+            var v = GetType().Assembly.GetName().Version.ToString();
+            var envi = Environment.GetEnvironmentVariable("ASPNETCORE_ENVIRONMENT");
+            DateTime utcTime = DateTime.UtcNow;
+            string localTime = utcTime.ToLocalTime().ToString("zzz");
+            string now = utcTime.ToString($"yyyy-MM-dd HH:mm:ss{localTime}");
```

Comment:
combine UTC date/time with the timezone of the local. I don't think it's correct. 🤔